### PR TITLE
Test file suffix determines non-bonded style.

### PR DESCRIPTION
**Pull request by ctk3b**
This could probably be implemented more cleanly eventually but it seems to work for now.

The differentiation for gromacs and desmond is done in `convert.py`, lammps is somewhat annoying because it needs to be defined in each test file. We could just import a setting file but some `atom_styles` change the way `pair_styles` are defined so I'm going to leave it manual for now and open an issue to remind myself to make this simpler.

### CODE REVIEW

File: intermol/convert.py [hunk 1]
```diff
@@ -178,12 +178,16 @@ def main(args=None):
...
         if args.get('gro_in'):
+            if gro_in.endswith('_vacuum.gro'):
+                mdp_path = os.path.abspath(os.path.join(tests_path, 'gromacs', 'grompp_vacuum.mdp'))
             input_type = 'gromacs'
             e_in, e_infile = gromacs_driver.gromacs_energies(top_in, gro_in, mdp_path, gropath, '')
         elif args.get('lmp_in'):
             input_type = 'lammps'
             e_in, e_infile = lammps_driver.lammps_energies(lammps_file, lmppath=lmppath)
         elif args.get('des_in'):
+            if cms_file.endswith('_vacuum.gro'):
```
Comment (mrshirts) [inline intermol/convert.py]: _vacuum.cms instead of vacuum.gro

File: intermol/lammps/lammps_parser.py [hunk 2]
```diff
@@ -1203,9 +1199,13 @@ def write(self, unit_set='real'):
 
             # non-bonded
             if atom_charges:
-                f.write('pair_style lj/cut/coul/cut 9.99999 19.99999\n')  # TODO: match mdp
```
Comment (mrshirts) [inline intermol/lammps/lammps_parser.py]: I'll play around with these settings (plus the gromacs_vacuum, onepoint.cfg settings.

File: intermol/lammps/lammps_parser.py [hunk 3]
```diff
@@ -1203,9 +1199,13 @@ def write(self, unit_set='real'):
...
+                if self.in_file.endswith('_vacuum.input'):
+                    f.write('pair_style lj/cut/coul/long 9.99999 19.99999\n')
```
Comment (mrshirts) [inline intermol/lammps/lammps_parser.py]: We probably want to make this longer (but I'll fiddle with this).  Will probably involve making the boxes bigger as well.

**Comment by ctk3b** [comment]
I'm going to be gone for a few hours but can make a little time later tonight if anything else needs doing before tomorrow.

**Pull request merged**